lambda/core/directinvoke/directinvoke.go

```diff
@@ -51,7 +52,7 @@ var ResetReasonMap = map[string]fatalerror.ErrorType{
 	"timeout": fatalerror.SandboxTimeout,
 }
 
-var MaxDirectResponseSize int64 = interop.MaxPayloadSize // this is intentionally not a constant so we can configure it via CLI
+var MaxDirectResponseSize = int64(interop.MaxPayloadSize) // this is intentionally not a constant so we can configure it via CLI
```

Comment:
Why do we have to cast it every time instead of only when setting it?

Reply:
as discussed, this is due to the change of the constant into a var that is used as both an int and int64 downstream